spec/requests/api/internal/v2/hearing_event_logs_request_spec.rb

```diff
@@ -63,22 +63,28 @@
         end
       end
 
-      context "when no hearing log is found" do
+      context "when no Hearing ID found" do
         let(:hearing_id) { "c6cf04b5" }
         let(:date) { "2020-02-09" }
 
         before do
           stub_request(:get, "#{ENV['COMMON_PLATFORM_URL']}/hearing/hearingLog?hearingId=#{hearing_id}&date=#{date}")
             .to_return(
-              status: 200,
```

Comment:
CP Api returns 500 when the Hearing is not found (not 200).
Note: this test used to work because it also returns an empty body.